frontend/app/src/components/settings/general/LanguageSetting.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <settings-option
+    #default="{ error, success, update }"
+    setting="language"
+    frontend-setting
+    :error-message="$t('general_settings.validation.language.error')"
+  >
+    <v-select
+      v-model="language"
+      :items="supportedLanguages"
+      item-text="label"
+      item-value="identifier"
+      outlined
+      :label="$t('general_settings.labels.language')"
+      persistent-hint
+      :success-messages="success"
+      :error-messages="error"
+      @change="update"
+    />
+  </settings-option>
```

Comment:
Maybe it would be helpful to add a hint here on how someone can contribute (we can create a guide for that and it would be nice if we can find an application that helps)

Also in a new PR maybe we can add Unicode flag icons in the language selector, we already have some code in the website